addons/website_payment/views/res_config_settings_views.xml

```diff
@@ -18,7 +18,12 @@
                             <div class="content-group">
                                 <div class="row mt8 ml-4" attrs="{'invisible': [('acquirers_state', '=', 'other_than_paypal')]}">
                                     <field name="acquirers_state" invisible="1"/>
-                                    <button name="action_activate_stripe" type="object" string="Activate Stripe" class="btn-primary"/>
+                                    <field name="is_stripe_supported_country" invisible="1"/>
+                                    <button attrs="{'invisible': [('is_stripe_supported_country', '=', False)]}"
+                                            name="action_activate_stripe" type="object" string="Activate Stripe" class="btn-primary"/>
+                                    <button attrs="{'invisible': [('is_stripe_supported_country', '=', True)]}"
+                                            name="action_activate_stripe" type="object" string="Activate Stripe" class="btn-primary disabled"
```

Comment:
Shouldn't it rather be:
```suggestion
                                            name="action_activate_stripe" type="object" string="Activate Stripe" class="btn-primary" disabled="true"
```
so that the button is truly disabled?

Reply:
Indeed

Reply:
Actually if we completely disable it we loose the tooltip.
I guess we are better off keeping like it is.

Reply:
Oh right. It's worth trying [something like this](https://stackoverflow.com/a/19938049) then.

Reply:
It does work but the tooltip has a different styling.
IMO it's not chocking.